lib/thrift/Kconfig

```diff
@@ -14,6 +14,15 @@ config THRIFT
   help
     Enable this option to support Apache Thrift
 
+config THRIFT_SSL_SOCKET
+  bool "Enable TSSLSocket support for Thrift"
+  depends on THRIFT
+  select MBEDTLS
```

Comment:
Oh - can you modify this slightly?

```
      Enable this option to support Apache Thrift

if THRIFT
...
depends on MBEDTLS
depends on MBEDTLS_PEM_CERTIFICATE_FORMAT
depends on NET_SOCKETS_SOCKOPT_TLS
...
     Enable this option to support TZlibTransport for Thrift

endif # THRIFT
```
`THRIFT` is implicit due to the `if THRIFT` above